.circleci/config.yml

```diff
@@ -34,3 +161,48 @@ jobs:
           name: Execute test
           command: ./test.sh flux
           no_output_timeout: 1500s
+
+workflows:
+  version: 2.1
+  on_push:
+    jobs:
+      - export_vars
+      - changelog:
+          filters:
+            branches:
+              only: /^1(.\d+)*$/
+      - test:
+          requires:
+            - export_vars
+      - fluxtest:
+          requires:
+            - export_vars
+  daily:
+    triggers:
+      - schedule:
+          # run weekdays at 4am -- note: use spaces, not tabs
+          cron: "0 4 * * 1-5"
+          filters:
+            branches:
+              only:
+                - "master-1.x"
+                - "1.8"
+                - "1.7"
```

Comment:
We did recently release a 1.7, but hopefully have no more releases on that branch.

Reply:
Ah ok. I can omit that branch.